packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -1,631 +1,249 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import { LitElement, html, render } from 'lit';
-import { customElement, property, query } from 'lit/decorators.js';
-import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { LitElement, html } from 'lit';
+import { customElement, property, state, queryAll } from 'lit/decorators.js';
+import { MediaQueryController } from '@vaadin/component-base/src/media-query-controller.js';
 import '@conversionxl/cxl-lumo-styles';
 import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
-import { MediaQueryController } from '@vaadin/component-base/src/media-query-controller.js';
-import { throttle } from 'lodash-es';
 import cxlMarketingNavStyles from '../styles/cxl-marketing-nav-css.js';
 import cxlMarketingNavGlobalStyles from '../styles/global/cxl-marketing-nav-css.js';
-import '@vaadin/button';
-import '@vaadin/tabs';
-import '@vaadin/context-menu';
+import '@vaadin/icon';
+import '@vaadin/menu-bar';
 
 @customElement('cxl-marketing-nav')
 export class CXLMarketingNavElement extends LitElement {
-  @property({ type: Object })
-  contextMenuItems;
-
-  @property({ type: Boolean, reflect: true })
-  fixed = false;
-
-  @query('.menu-items')
-  menuShadowItemsElement;
-
-  @property({ type: HTMLElement })
-  get menuItemSearchElement() {
-    return this.shadowRoot.querySelector('.menu-item-search');
-  }
-
-  @property({ type: NodeList })
-  get menuItemsElements() {
-    return this.querySelectorAll('.menu-items');
+  static get styles() {
+    return [cxlMarketingNavStyles];
   }
 
-  @property({ type: Number })
-  menuShadowItemsSelectedIdx = -1;
-
-  // Conditionally hide distracting shadow menu items. Used for checkout, etc.
-  @property({ type: Boolean, reflect: true })
-  minimal = false;
+  _contextMenuItems = { global: [], primary: [] };
 
-  // MediaQueryController.
-  @property({ type: Boolean, reflect: true })
-  wide;
+  _phoneMediaQuery = '(max-width: 568px), (max-height: 568px)';
```

Comment:
This is possibly incorrect: https://app.clickup.com/t/86aycmmmx?comment=90140006129795

Reply:
Not for the new element, but there are other things to deal with regarding the mobile layout. Working on it.